packages/core/src/character/controller-remote.ts

```diff
@@ -79,6 +79,7 @@ export class RemoteController {
   }
 
   transitionToAnimation(targetAnimation: string, transitionDuration: number = 0.21): void {
+    console.log(targetAnimation);
```

Comment:
Rogue `console.log`

Reply:
Removed from this line in [commit 5](https://github.com/mml-io/mml-playground/pull/21/commits/01fc1ea646d4c1e9d0eb55bcff3e2de0202b47ff#diff-9c4c97202846464e0af78e2c47bf309132f8fe1efb4f62291e22c03d8b85577bL82)